components/script/dom/webglframebuffer.rs

```diff
@@ -108,16 +116,36 @@ impl WebGLFramebuffer {
             stencil: DomRefCell::new(None),
             depthstencil: DomRefCell::new(None),
             is_initialized: Cell::new(false),
+            xr_session: Default::default(),
         }
     }
 
     pub fn maybe_new(context: &WebGLRenderingContext) -> Option<DomRoot<Self>> {
         let (sender, receiver) = webgl_channel().unwrap();
         context.send_command(WebGLCommand::CreateFramebuffer(sender));
-        receiver
-            .recv()
-            .unwrap()
-            .map(|id| WebGLFramebuffer::new(context, id))
+        let id = receiver.recv().unwrap()?;
+        let framebuffer = WebGLFramebuffer::new(context, WebGLFramebufferId::Transparent(id));
+        Some(framebuffer)
+    }
+
+    // TODO: depth, stencil and alpha
```

Comment:
Is this still true? Are we regressing the rollercoaster depth fixes?

Reply:
It's complicated... the problem is that surfman considers depth/stencil/alpha on surfaces to be the same on all surfaces in the same context, and WebXR thinks of them as being independent. So this isn't spec-compliant, in that if the canvas has different settings for depth/stencil/alpha than the WebXR content, we're not going to respect them.

Shall I file an issue for this?

Reply:
Yes, please.

Reply:
https://github.com/servo/servo/issues/24498